lib/src/reducers/strands_reducer.dart

```diff
@@ -211,19 +211,26 @@ Strand move_strand(
       int delta_offset,
       bool delta_forward}) {
   List<Substrand> substrands = strand.substrands.toList();
+  var list_of_domains = [...strand.domains];
```

Comment:
Is this a way to make a copy of the list `strand.domains`? Somehow this would seem more "canonical" to me (or at least it's more to me clear what it's doing):

```dart
var list_of_domains = List<Domain>.from(strand.domains);
```

The `[...list]` notation is intended to make it easier to include lists inline in a long list of items, like when specifying React components, e.g., 

```dart
return [
  first_component(),
  ...list_of_components(),
  last_component(),
]
```

Also, I tend not to use the [Hungarian notation](https://en.wikipedia.org/wiki/Hungarian_notation) unless there are two different types of collections with the same object and I want to know which is which, i.e., if both `domains_set` and `domains_list` were in the same scope. But when it's just one collection, I usually just use the plural such as `domains`. In particular, List is so much more common than Set that `domains` almost always for me refers to a list of domains.